<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QDialog" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>462</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(171, 216, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="4,1">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
            <item>
             <widget class="QToolButton" name="Host_bt">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>SimSun-ExtB</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Host</string>
              </property>
              <property name="icon">
               <iconset resource="../image.qrc">
                <normaloff>:/new_ui/UI/04-ScreenCalibration.png</normaloff>:/new_ui/UI/04-ScreenCalibration.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>72</width>
                <height>72</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="RTDATA_bt">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Rt.Data</string>
              </property>
              <property name="icon">
               <iconset resource="../image.qrc">
                <normaloff>:/new_ui/UI/03-RealTimePower.png</normaloff>:/new_ui/UI/03-RealTimePower.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>72</width>
                <height>72</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
            <item>
             <widget class="QToolButton" name="Record_bt">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Record</string>
              </property>
              <property name="icon">
               <iconset resource="../image.qrc">
                <normaloff>:/new_ui/UI/03-HistoricalRecord.png</normaloff>:/new_ui/UI/03-HistoricalRecord.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>72</width>
                <height>72</height>
               </size>
              </property>
              <property name="autoRepeatDelay">
               <number>298</number>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="System_bt">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>System</string>
              </property>
              <property name="icon">
               <iconset resource="../image.qrc">
                <normaloff>:/new_ui/UI/06-SystemSettings.png</normaloff>:/new_ui/UI/06-SystemSettings.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>72</width>
                <height>72</height>
               </size>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextUnderIcon</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="Open_bt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Turn on</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/new_ui/UI/PowerOn-01.png</normaloff>:/new_ui/UI/PowerOn-01.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="Standby_bt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Stand-by</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/new_ui/UI/Power_Standby.png</normaloff>:/new_ui/UI/Power_Standby.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="Close_bt">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Turn off</string>
         </property>
         <property name="icon">
          <iconset resource="../image.qrc">
           <normaloff>:/new_ui/UI/PowerOff-02.png</normaloff>:/new_ui/UI/PowerOff-02.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
